frontend/tests/controllers/CoderOfTheMonthTest.php

```diff
@@ -324,7 +328,7 @@ private function getCoderOfTheMonth($revDate, $interval) {
         );
         $response = \OmegaUp\Controllers\User::apiCoderOfTheMonth(
             new \OmegaUp\Request(
-                []
+                [ 'category' => $category]
```

Comment:
nit:

```suggestion
                ['category' => $category]
```

no deben haber espacios entre los corchetes / paréntesis y sus contenidos.

Reply:
listo